docker/init_scripts/init

```diff
@@ -5,10 +5,6 @@ set -o nounset           # treat unset variables as an error
 set -o pipefail          # treat errors in pipes as fatal
 shopt -s inherit_errexit # inherit errexit
 
-# use virtualenvs
-# shellcheck disable=SC1091
-source /backend/bin/activate
```

Comment:
will be automatically sourced with this change?

Reply:
With the change to the `PATH` environment variable, the virtualenv basically becomes the "official" Python environment inside the Docker container, so there's no need to activate/deactivate. We get the isolation of the venv approach, without the need to remember to call the appropiate `python`/`pip` binary!